core/git-access.c

```diff
@@ -359,10 +359,10 @@ static int try_to_git_merge(git_repository *repo, git_reference **local_p, git_r
 	}
 
 	git_merge_init_options(&merge_options, GIT_MERGE_OPTIONS_VERSION);
-#if !LIBGIT2_VER_MAJOR && LIBGIT2_VER_MINOR > 23
-	merge_options.flags = GIT_MERGE_FIND_RENAMES;
-#else
+#if !LIBGIT2_VER_MAJOR && LIBGIT2_VER_MINOR < 22
```

Comment:
So this is obviously incorrect as it switches from `>23` to in effect `!<22` which is very wrong.
But the bigger question is "why would we still support anything older than libgit2 0.26.
I'll need to go through the source and make that happen.

Reply:
There's only 3 occurences - I can update the PR to get rid of the old usages.